bower.json

```diff
@@ -20,7 +20,8 @@
   "homepage": "https://github.com/AqoviaElements/epi-cms-property",
   "ignore": [],  
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.4.0"
+    "polymer": "Polymer/polymer#^1.4.0",
+    "properties-from-ancestor-behavior": "~1.0"
```

Comment:
Well internally the major version number is not controlled in source. Here it's in source, so I think we should take it upon ourselves to always consider breaking changes - and have adequate tests to identify them. But I see that it merits discussion :-)